### ISSUE TITLE
feature: Generate UUID without prefix "urn:uuid:"

### ISSUE THREAD

**Issue by usmanovbf** [stoplightio/prism#365]
**Is your feature request related to a problem? Please describe.**
It's inconvenient to parse by clients mocks from example where attribute with uuid format could be like  "urn:uuid:df48da63-0b3e-2392-f080-a0350cb1a100"  or "df48da63-0b3e-2392-f080-a0350cb1a100" 
**Describe the solution you'd like**
Just leave "df48da63-0b3e-2392-f080-a0350cb1a100"  format

**Comment by philsturgeon** [comment]
Sorry @usmanovbf but I really do not know what this means. 

Could you maybe provide some sort of example to elaborate your point?

**Comment by philsturgeon** [comment]
@usmanovbf could you provide some information so we can move on this bug? What is the spec you are using? What command are you running? What version are you using?

We have an issue template for bugs, which includes these questions. The goal is to help you provide enough information to help us. 

```
**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**

Using this example specification:

``` yaml
openapi: 3.0
# ...
```

Steps to reproduce the behavior:

1. When I run this command: `prism mock example.yaml --foo --bar`
2. Then make the following HTTP request: `curl GET ...`
3. Then something unexpected happens...

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. macOS 10.14.4]
- Prism Version [e.g. v3.0.0]
- Install method [e.g. binary, npm, yarn, docker]
```

**Comment by andresblancomorales** [comment]
I think I know what he means.

When setting a property to type `string` and format `uuid` and running prism with `-d` flag, sometimes it generates a uuid like: 207d1180-9659-cec6-6a6d-05d1a5e6f830 but sometimes it generates the uuid prefixed with `urn:uuid:` like: urn:uuid:207d1180-9659-cec6-6a6d-05d1a5e6f830

**Comment by andresblancomorales** [comment]
It looks like it's generated by the `json-schema-faker` library.

You can see it [here](https://github.com/json-schema-faker/json-schema-faker/blob/master/src/generators/coreFormat.js)

```
uuid: '^(?:urn:uuid:)?[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$'
```

**Comment by XVincentX** [comment]
We have just released 3.2.0-beta.5 that fixes this issue. You can try it out installing Prism from npm with its beta tag

`npm i @stoplight/prism-cli@beta`
